packages/dds/tree/src/feature-libraries/node-key/nodeKeyManager.ts

```diff
@@ -90,50 +90,3 @@ export function createNodeKeyManager(idCompressor?: IIdCompressor | undefined):
 export function isStableNodeKey(key: string): key is StableNodeKey {
 	return isStableId(key);
 }
-
-/**
- * Create a {@link NodeKeyManager} that generates deterministic {@link StableNodeKey}s and {@link LocalNodeKey}s.
- * @remarks This is useful for test environments because it will always yield the same keys in the same order.
- * It should not be used for production environments for the same reason; the {@link StableNodeKey}s are not universally unique.
- */
-export function createMockNodeKeyManager(): MockNodeKeyManager {
-	return new MockNodeKeyManager();
-}
-
-class MockNodeKeyManager implements NodeKeyManager {
```

Comment:
I believe this was here (not under test) so that users of our package could do deterministic unit testing of their apps. Given this isn't package exported and isn't at the abstraction level our public API uses, I don't think its possible for people to actually use it, but we may want to conder how we want to enable users of tree and identifiers too do deterministic testing. Will that require using this code (indirectly)?

Reply:
I can revert this if we prefer, then. Is there anything special about this implementation that a customer couldn't do themselves?